src/plone/restapi/types/utils.py

```diff
@@ -76,13 +76,20 @@ def get_ordered_fields(fti):
     return (fields, ordered_fieldsets_fields)
 
 
+def get_hidden_fields_from_schema(schema):
```

Comment:
@lukasgraf you are reviewing faster than I can type. :) Good point. I will look into that if we figured out the more general issues (adapter pattern vs tagged values).